docker/nginx.conf

```diff
@@ -9,7 +9,7 @@ http {
 
     include /etc/nginx/mime.types;
 
-    proxy_cache_path /cache/nginx levels=1:2 keys_zone=takahe:20m inactive=14d max_size=__CACHESIZE__;
+    proxy_cache_path /cache/nginx levels=1:2 keys_zone=takahe:20m inactive=14d max_size=1g;
```

Comment:
Why did you remove the dynamic cache size replacement? This should be configurable.